config.js

```diff
@@ -13,17 +13,24 @@ System.config({
       "deps": [
         "jquery"
       ]
+    },
+    "showdown/dist/showdown.js": {
+      "format": "global"
     }
   },
   map: {
     "aurelia-after-attached-plugin": "github:aurelia-ui-toolkits/aurelia-after-attached-plugin@0.1.0",
     "aurelia-animator-css": "npm:aurelia-animator-css@1.0.1",
     "aurelia-bootstrapper": "npm:aurelia-bootstrapper@1.0.1",
+    "aurelia-event-aggregator": "npm:aurelia-event-aggregator@1.0.1",
```

Comment:
`jspm install aurelia-event-aggregator` is necessary as we use `import {EventAggregator} from 'aurelia-event-aggregator'`. If you do not, SystemJS will think `aurelia-event-aggregator` is an absolute url and will try to load `src/aurelia-event-aggregator.js` which results in a 404